src/resources/i18n/translations/en.json

```diff
@@ -568,11 +565,11 @@
 	"The email may go to your Spam folder": "The e-mail may be in you spam folder.",
 	"The start time must be before the end time": "Start time must be before the end time.",
 	"The start time must be greater than 0 if no end time is provided": "Start time must be above 0 if no end time is provided.",
-	"Third Party Libraries": "Libre libraries",
+	"Third Party Libraries": "Third Party Libraries",
```

Comment:
```suggestion
	"Third Party Libraries": "Third-Party Libraries",
```

Reply:
👍 I'm not sure how we track it separately with Weblate though.

The podverse-fdroid repo is a fork of podverse-rn, so it seems like it might be overkill to add the whole repo to Weblate. Is there a way to note within the podverse-rn Weblate that alternate translation is needed for F-Droid?

Reply:
If there is an extra string available, it could be translated.

Reply:
Just made a commit with "Libre Libraries" to this branch.